gradle/libs.versions.toml

```diff
@@ -50,7 +50,7 @@ rulesVersion = "1.5.0"
 runnerVersion = "1.5.2"
 securityCryptoKtx = "1.1.0-alpha06"
 splashScreenVersion = "1.0.1"
-targetSdk = "34"
+targetSdk = "35"
```

Comment:
we need to update complieSdk version as well to 35. also our current AGP version is not compatible with target sdk 35, and we should update it as well, you can update to `agp = "8.10.1"`